lib/aiko/led.py

```diff
@@ -62,7 +64,6 @@
 yellow = colors["yellow"]
 
 def apply_dim(color, dimmer=None):
-  global dim
```

Comment:
Oh, good point. We don't want to change it, so we don't need this.

Reply:
indeed :)